geoindicators/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/SpatialUnitsTests.groovy

```diff
@@ -197,4 +209,30 @@ class SpatialUnitsTests {
         assert 246 == countRows.numberOfRows
     }
 
+    @EnabledIfSystemProperty(named = "test.h2gis", matches = "false")
+    @Test
+    void regularGridTestH2GIS() {
+        def gridP = Geoindicators.SpatialUnits.regularGrid()
+        def wktReader = new WKTReader()
+        def box = wktReader.read('POLYGON((-180 -80, 180 -80, 180 80, -180 80, -180 -80))')
```

Comment:
It would be nice to reduce the bounding box area for the test.

Reply:
and check if the have polygon geometries as output

Reply:
I made the test with a smaller bounding area, containing 100 points